src/index.js

```diff
@@ -412,7 +412,7 @@ export function check(pretty = true) {
 		}
 
 		// Check if default is set
-		if (opts.default) {
+		if (opts.default != null) {
```

Comment:
Yeah, removing the key from `defaultOpts` would let us use the `in` to check.

One issue I'm thinking of now is the other potential landmine of usability. If I set an env-var to `null`, it'll resolve as `process.env[KEY] === "null"`, which could be an unexpected result for someone who didn't know better and previously set a default to `null`. If their code did a simple `if (ENV_VAR) { ... }` check, it'd suddenly start being true by default.

I mention it because someone did that in our code base, and this change would have broken a module.